macos/FluentUI/FilledTemplateImageView/FilledTemplateImageView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+/// Custom view for displaying a template style image that can be filled with a specified color. This takes in an image with a corresponding
+/// overlay mask that is used for drawing the fill. The image and its corresponding fillMask need to line up appropriately so that when
+/// the image is drawn over its mask, its outline will perfectly overlap the edges of the mask to achieve the desired fill effect.
+@objc(MSFFilledTemplateImageView)
+open class FilledTemplateImageView: NSImageView {
+
+	/// Creates a custom template style image view with a fill.
+	/// - Parameters:
+	///   - image: the template style image or icon to be drawn.
+	///   - fillMask: the mask image used to draw the fill color.
+	///   - contentTintColor: the color to use for the main image outline.
+	///   - fillColor: the color to use for the image fill. When set to nil or clear, the fill won't be drawn.
+	@objc(initWithImage:fillMask:contentTintColor:FillColor:)
+	public init(
+		image: NSImage,
+		fillMask: NSImage,
+		contentTintColor: NSColor,
+		fillColor: NSColor
```

Comment:
You implied in the documentation that this could be set to `nil`, but you did not declare it as optional (`NSColor?`).

Reply:
No, I was torn on that one actually I've grown. Seemed better to not let clients initialize with nil initially (what's the point of using this image with no fill). Then I thought about it and perhaps they want to set it to no fill initially then fill it later - which seems like a legit case.